FutureNorthants/northants-website
# Return redirect if path begins with /node/

Check if the path begins with `/node/` and return a DrupalRedirectRoute with the alias set as the destination. 

Additionally check that there is an alias set or that the alias is not the same as the path, otherwise it could cause a redirect loop. 

## To Test

- View a page in the CMS and make note of the url
- Press edit on a page in the CMS and then make note of the node id in the url
- Use the below graphql query and replace the 169 with the node id
- Ensure that the destination matches the url in the view page in the CMS
- Load the '/node/169' url (or whatever you have tested) in the frontend and it should be redirected to the destination

```graphql
{
  route(path: "/node/169") {
    __typename
    ... on DrupalRedirectRoute {
       __typename
      destination
      status
    }
  }
}
```
